modules/drivers/usb_cam_new/src/LICENSE

```diff
@@ -0,0 +1,14 @@
+Video for Linux Two API Specification
+Revision 0.24
+Michael H Schimek  <mschimek@gmx.at>
+Bill Dirks
+Hans Verkuil
+Martin Rubli
```

Comment:
If this code is from another package, can we refer to the code by a library dependency?